frontend/src/components/Agreements/AgreementMetaAccordion/AgreementMetaAccordion.jsx

```diff
@@ -57,6 +57,19 @@ const AgreementMetaAccordion = ({ agreement, projectOfficerName, res, cn, conver
                                 className={cn("type")}
                                 value={convertCodeForDisplay("agreementType", agreement?.agreement_type)}
                             />
+                            <Term
+                                name="contract-type"
```

Comment:
🤔 Does this need validation?

Reply:
what a minimal Agreement looks like with current validation

<img width="925" alt="image" src="https://github.com/user-attachments/assets/fe7906f9-6d46-49fa-818d-6c401ec6753b">

Reply:
**Service Requirement Type** is a mandatory field when creating an agreement

Reply:
will add validation